serviceconnector/src/deleteServiceConnector/deleteLinker.ts

```diff
@@ -0,0 +1,40 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.md in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+
+import { AzExtTreeItem, AzureWizard, AzureWizardExecuteStep, AzureWizardPromptStep, IActionContext, createSubscriptionContext } from "@microsoft/vscode-azext-utils";
+import * as vscode from 'vscode';
+import { LinkerItem } from "../createServiceConnector/createLinker";
+import { DeleteLinkerStep } from "./DeleteLinkerStep";
+import { IPickLinkerContext } from "./IPickLinkerContext";
+import { LinkerListStep } from "./LinkerListStep";
+
+export async function deleteLinker(context: IActionContext, item: LinkerItem | AzExtTreeItem): Promise<void> {
+    const subscription = item instanceof AzExtTreeItem ? item.subscription : createSubscriptionContext(item.subscription);
+
+    const wizardContext: IPickLinkerContext = {
+        ...context,
+        ...subscription,
+        sourceResourceUri: item?.id,
+    }
+
+    const title: string = vscode.l10n.t('Delete service connector');
```

Comment:
`title` is only used when you create the wizard down below so I'd just eliminate the variable.